mungegithub/publisher/README.md

```diff
@@ -26,17 +26,17 @@ The robot is also responsible to update the `Godeps/Godeps.json` and the `vendor
 
 Currently we don't have tests for the robot. It relies on manual tests:
 
-* Fork the repos you are going the publish. Run [fetch-all-latest-and-push.sh](util/fetch-all-latest-and-push.sh) to update the branches of your repos.
+* Fork the repos you are going the publish. Run [fetch-all-latest-and-push.sh](fetch-all-latest-and-push.sh) to update the branches of your repos.
 
-* Change `config.organization` to your github username in `mungegithub/publisher/deployment/kubernetes/configmap.yaml`
```

Comment:
This structure was for consistency with other deployments. Moving the configmap will break `make push_config`.